apps/src/code-studio/pd/foorm/editor/FoormEditorHeader.jsx

```diff
@@ -0,0 +1,165 @@
+// Metadata and buttons for the Foorm Editor. Shows form name and version,
```

Comment:
nothing in this file is new--content moved from FoormEditor and validate button moved below live preview